src/services/posts/fetch-posts.js

```diff
@@ -1,8 +1,8 @@
 const db = require("../../db")
 
-const fetchPosts = async(isFlagged) => {
+const fetchPosts = async(isFlagged, cursor, limit) => {
```

Comment:
i missed something here in our discussion, cursor will not be always available, especially for the first requests. 

the whereClause which has the cursor should only be used if cursor is defined

Reply:
alright, i will work on that too